### ISSUE TITLE
[BUG] - Configure conda-store allowed channels via nebari config

### ISSUE THREAD

**Issue by alimanfoo** [nebari-dev/nebari#1594]
### Describe the bug

I'd like to configure the [CondaStore.conda_allowed_channels](https://conda.store/en/latest/administration.html?highlight=channel#conda-store-server-app-condastore) via the nebari configuration file. This should be possible as described [here](https://github.com/nebari-dev/nebari/discussions/1567#discussioncomment-4267160):

```
conda_store:
  extra-settings:
     CondaStore:
       conda_allowed_channels = []
```

However, this does not validate according the nebari scheme, which requires "extra_settings". 

Trying this configuration instead:

```
conda_store:
  extra_settings:
     CondaStore:
       conda_allowed_channels = []
```

...does validate but may result in a broken conda-store service.

### Expected behavior

In general any configuration parameter should be possible to pass through to conda-store via nebari-config.yaml.

### OS and architecture in which you are running Nebari

GKE

### How to Reproduce the problem?

Try the example configs given above and deploy.

### Versions and dependencies used.

Nebari version 2022.11.1.

### Compute environment

GCP

### Integrations

conda-store

### Anything else?

Originally discussed here: https://github.com/nebari-dev/nebari/discussions/1567.

**Comment by iameskild** [comment]
Thanks @alimanfoo for opening this! I haven't had a chance to test this yet. I will report back shortly :)

**Comment by pavithraes** [comment]
@iameskild Thank you for taking a look, pinging just to surface this for you. :)

**Comment by costrouc** [comment]
@alimanfoo I know this is a while back but would it be possible for you to share the Kubernetes logs for conda-store-server and conda-store-worker? I agree that this should not be crashing with this setting.

**Comment by viniciusdc** [comment]
Hi @alimanfoo. Thanks for reporting this. If I understood correctly, you got a problem with the `extra-config` not being a standard key for conda_store, right? Have you encountered any issues with using `extra_config` instead, if so, could you post them as @costrouc commented above?

Just give some context, the reason why `extra-config` failed is mostly probably dues to a typo here:
https://github.com/nebari-dev/nebari/blob/49bca7153d14965f06340bf50f24e054fc0ca4d7/nebari/stages/input_vars.py#L288-L292
and here:
https://github.com/nebari-dev/nebari/blob/49bca7153d14965f06340bf50f24e054fc0ca4d7/nebari/schema.py#L125-L126
It should be `extra-setting` and `extra-config` to comply with conda-store docs. 

This also means that for Nebari (at least the current versions), the standard key will be `extra_config/extra_settings`. They should not break conda-store because they are correctly passed later to conda-store with the correct syntax here:
https://github.com/nebari-dev/nebari/blob/49bca7153d14965f06340bf50f24e054fc0ca4d7/nebari/template/stages/07-kubernetes-services/conda-store.tf#L63-L64

I do agree that having those different structures to the same keys are confusing and unnecessary, so an action item would be to rename that to the correct syntax as per conda-store docs

**Comment by alimanfoo** [comment]
Hi folks, thanks so much for following this one up, much appreciated. Just wanted to say apologies I'm unlikely to have time to do more on this in the short term and I'm afraid logs from when I tried this before will be long gone having been through a number of cycles of destroying and redeploying clusters since.

**Comment by dharhas** [comment]
@viniciusdc @iameskild @costrouc I need to do the same thing and it is pretty hard to parse from the conversation above what actually needs to be done to allow other channels. We urgently need this on demo.nebari.dev
